### ISSUE TITLE
10.0 business_requiremente_deliverable module Roadmap for pricelist calculation

### ISSUE THREAD

**Issue by martintorresvm** [OCA/business-requirement#256]
Remove function `_get_pricelist` and all it calls:
https://github.com/OCA/business-requirement/blob/10.0/business_requirement_deliverable/models/business.py#L227

Add on business.requirement module
```
pricelist_id = many2one with these properties:
comodel_name='product.pricelist',
required = True
default = `default_pricelist`

@api.multi
def default_pricelist(self):
    for br in self:
         return (br.partner_id.property_product_estimation_pricelist or
                    br.partner_id.property_product_pricelist)
```

Check all methods we called before to the function  `_get_pricelist`:

 1. Now should be get the value from the pricelist_id so on `business.requirement.deliverable` model we need to get from the `self.business_requirement_id.pricelist_id`
 2. Specific for the `computed` methods is add inside decorator `depends` this `business_requirement_id.pricelist_id`

Updated:
If there is no pricelist found, the DL or RL sales prices should be the product sales price.

**Comment by martintorresvm** [comment]
@elicoidal Please have a check and if correct add to the milestone to v10.0

**Comment by elicoidal** [comment]
Partner should not be mandatory and neither the pricelist.
If there is no pricelist found, the DL or RL sales prices should be the product sales price.

**Comment by elicoidal** [comment]
@victormartinelicocorp

**Comment by martintorresvm** [comment]
> Partner should not be mandatory and neither the pricelist.
If there is no pricelist found, the DL or RL sales prices should be the product sales price.

@elicoidal I will amend it.

**Comment by elicoidal** [comment]
:+1:

**Comment by elicoidal** [comment]
fixed by #225
